Syn/JwtAuthenticator.php

```diff
@@ -72,57 +63,44 @@ public function getCredentials(Request $request)
             $jwt = $this->jwtFactory->load($token);
         } catch (InvalidArgumentException $exception) {
             $this->logger->info('Invalid token. ' . $exception->getMessage());
-            return null;
+            return [
+              'token' => $token,
+              'name' => null,
+              'roles' => null,
+            ];
         }
 
         // Check correct properties
         $payload = $jwt->getPayload();
-        if (!isset($payload['webid'])) {
-            $this->logger->info('Token missing webid');
-            return null;
-        }
-        if (!isset($payload['iss'])) {
-            $this->logger->info('Token missing iss');
-            return null;
-        }
-        if (!isset($payload['sub'])) {
-            $this->logger->info('Token missing sub');
-            return null;
-        }
-        if (!isset($payload['roles'])) {
-            $this->logger->info('Token missing roles');
-            return null;
-        }
-        if (!isset($payload['iat'])) {
-            $this->logger->info('Token missing iat');
-            return null;
-        }
-        if (!isset($payload['exp'])) {
-            $this->logger->info('Token missing exp');
-            return null;
-        }
-
-        if ($jwt->isExpired()) {
-            $this->logger->info('Token expired');
-            return null;
-        }
 
         return [
             'token' => $token,
             'jwt' => $jwt,
-            'name' => $payload['sub'],
-            'roles' => $payload['roles']
+            'name' => isset($payload['sub']) ? $payload['sub'] : null,
+            'roles' => isset($payload['roles']) ? $payload['roles'] : null,
```

Comment:
Small stylistic thing... null coalescing instead of the ternary doing the same:

```suggestion
            'name' => $payload['sub'] ?? null,
            'roles' => $payload['roles'] ?? null,
```

... or... then again... might almost make sense to build up the array differently, if we always expect to be providing things in it?